.github/workflows/upload-lib.yml

```
@@ -32,3 +32,14 @@ jobs:
         conan remote add artifactory https://artifactory.matthewcotton.net/artifactory/api/conan/conan-local
         conan remote login artifactory ci -p ${{ secrets.ARTIFACTS_TOKEN }}
         conan upload moth_ui --confirm --remote artifactory
+
+    - run: Fetch package version
+      id: get-version
+      run: |
```

actionlint (1.7.4)

key "run" is duplicated in element of "steps" section. previously defined at line:36,col:7

(syntax-check)